Fix ping module with LANG fixed to C

tyjak opened the pull request:
Fix a pb when env language is different from LANG C

tyjak committed:
Fix ping module with LANG fixed to C

tyjak committed:
Fix ping module with LANG fixed to C

tyjak committed:
Fix and refactor bticoin module

* change API endpoint to blockchain.info
* remove wallet monitor functionality

tyjak committed:
Fix and refactor bticoin module

* change API endpoint to blockchain.info
* remove wallet monitor functionality

tyjak committed:
Apply MR #869

tyjak committed:
Apply MR #869

tyjak committed:
Fix battery status

see https://github.com/enkore/i3pystatus/issues/877

chestm007 commented:
This PR looks like it has accidentally pulled in additional changes - could you clean it up please and ill take another look!

tyjak committed:
Merge branch 'enkore:current' into current

tyjak committed:
Fix test suite

A series of fixes to get the test suite to pass on a modern Python environment.

- Remove notmuch to fix incompatibility with python 3.14.2 see issue #2
- Create a virtual environment for tests.
- Fix various pycodestyle errors.
- Fix PYTHONPATH to correctly locate the i3pystatus package.
- Upgrade Sphinx to a newer version to fix compatibility issues with Python 3.
- Add missing test dependencies (requests, pytz, setuptools).
- Fix reStructuredText formatting in docstrings.
- Fix an import issue in the custom Sphinx extension.

tyjak committed:
Merge pull request #1 from tyjak/test-suite-fix

Fix test suite

tyjak committed:
Fix remove mock dependencie

tyjak committed:
Merge pull request #3 from tyjak/fix-mock-dependencie-issue

Fix remove mock dependencie
